openslides/global_settings.py

```diff
@@ -47,6 +47,9 @@
     },
 ]
 
+# Email
```

Comment:
Please the following link: `https://docs.djangoproject.com/en/1.10/topics/email/` and newline.

Reply:
Done.